### PR TITLE
Refactor claim status files tab styles

### PR DESCRIPTION
**Pull request by jbalboni**
Pretty much what it says. Beyond the basic refactoring, I updated some of the alert boxes to use flexbox for the details button and adjusted/removed some of the `h` tags to make the hierarchy more reasonable.

### CODE REVIEW

**Review by cehsu — APPROVED** [review]
LGTM. There are a few comments and visual differences (I will add screenshots for these).

**Comment by cehsu** [comment]
Here are the visual differences, not sure if they are both deliberate:

### Tab styles on mobile
screen shot 2017-10-18 at 6 22 04 pm

### Padding/margin below "No longer needed"
screen shot 2017-10-18 at 6 19 20 pm

### Closer view of padding/margin below "No longer needed"
screen shot 2017-10-18 at 6 18 49 pm

File: src/js/claims-status/components/AddFilesForm.jsx [hunk 1]
```diff
@@ -87,7 +87,7 @@ class AddFilesForm extends React.Component {
           }}>Need to mail or fax your files</a>?</p>
         </div>
         <Element name="filesList"/>
-        <div className="button-container">
+        <div>
           <ErrorableFileInput
```
Comment (cehsu) [inline src/js/claims-status/components/AddFilesForm.jsx]: I am no longer able to test this locally, is that just on my end?

**Comment by jbalboni** [comment]
@cehsu The tab change was not intentional, that's from a typo. The other spacing change was intentional, since the spacing should be consistent whatever the status line in those cards is.

File: src/js/claims-status/components/AddFilesForm.jsx (hunk 1 above)
Comment (jbalboni) [reply src/js/claims-status/components/AddFilesForm.jsx]: You should still be able to do. Make sure you have the claims-mock file I sent you running, vets-api is not runnig, and there's a value in `sessionStorage.userToken` in the tab you're using.

File: src/js/claims-status/components/RequestedFilesInfo.jsx [hunk 2]
```diff
@@ -12,46 +12,49 @@ export default class RequestedFilesInfo extends React.Component {
...
-              <div className="clearfix"></div>
+              <div className="button-container">
+                <Link aria-label={`View Details for ${item.displayName}`} title={`View Details for ${item.displayName}`} className="usa-button usa-button-outline view-details-button" to={`your-claims/${id}/document-request/${item.trackedItemId}`}>View Details</Link>
+              </div>
             </div>
           ))}
 
           {optionalFiles.map(item => (
-            <div className="file-request-list-item usa-alert file-request-list-item-optional no-background-image" key={item.trackedItemId}>
+            <div className="file-request-list-item usa-alert file-request-list-item-optional no-background-image alert-with-details" key={item.trackedItemId}>
```
[the remark this answers is not in the document]
Comment (jbalboni) [reply src/js/claims-status/components/RequestedFilesInfo.jsx]: Looks ok to me locally.

**Pull request merged**